### PR TITLE
chore: Add dataId to the every menu items

### PR DESCRIPTION
**Pull request by HoonBaek**
### Description Of Changes

Add dataId to the every menu items: MenuItem and ReactionButton

### CODE REVIEW

**Review by sravan-s — APPROVED** [review]
lets use `data-sb-id` please

**Review by sravan-s — CHANGES_REQUESTED** [review]
lets call these `dataSbId` :) .. even tho data-ids are fine, these would reduce confusion with clients

File: src/modules/ChannelSettings/components/ModerationPanel/MemberList.tsx [hunk 1]
```diff
@@ -20,7 +21,7 @@ import { LocalizationContext } from '../../../../lib/LocalizationContext';
 import uuidv4 from '../../../../utils/uuid';
 
 export const MemberList = (): ReactElement => {
-  const [members, setMembers] = useState([]);
+  const [members, setMembers] = useState<Array<Member>>([]);
```
Comment (sravan-s) [inline src/modules/ChannelSettings/components/ModerationPanel/MemberList.tsx:24]: 👍

File: src/ui/MessageItemReactionMenu/index.tsx [hunk 2]
```diff
@@ -92,6 +92,7 @@ export default function MessageItemReactionMenu({
                       closeDropdown();
                       toggleReaction(message, emoji.key, isReacted);
                     }}
+                    dataSbId={emoji.key}
```
Comment (sravan-s) [inline src/ui/MessageItemReactionMenu/index.tsx:95]:
```
dataSbId={`reaction_menu_${emoji.key}`}
```
would be more trackable

File: src/ui/MobileMenu/MobileBottomSheet.tsx [hunk 3]
```diff
@@ -101,6 +101,7 @@ const MobileBottomSheet: React.FunctionComponent<MobileBottomSheetProps> = (prop
                           hideMenu();
                           toggleReaction(message, emoji.key, isReacted);
                         }}
+                        dataSbId={emoji.key}
```
Comment (sravan-s) [inline src/ui/MobileMenu/MobileBottomSheet.tsx:104]: something like
```
dataSbId={`emoji_bottom-sheet_${emoji.key}`}
```
would be more trackable

**Review by sravan-s — COMMENTED** [review]
Some minor comments:
For items in list -> 
`item-type_component-type_key` would be more trackable ->`emoji_bottom-sheet_laugh-1`

File: src/ui/MobileMenu/MobileBottomSheet.tsx (hunk 3 above)
Comment (HoonBaek) [reply src/ui/MobileMenu/MobileBottomSheet.tsx:104]: okay okay let me apply it

File: src/modules/ChannelList/components/ChannelPreviewAction.jsx [hunk 4]
```diff
@@ -60,6 +60,8 @@ export default function ChannelPreviewAction({
                 setShowModal(true);
                 closeDropdown();
               }}
+              dataSbId="channel_list_item_context_menu_leave_channel"
+
```
Comment (sravan-s) [inline src/modules/ChannelList/components/ChannelPreviewAction.jsx:64]: space ;)

File: src/modules/OpenChannelSettings/components/OperatorUI/BannedUsersModal.tsx [hunk 5]
```diff
@@ -104,6 +104,7 @@ export default function BannedUsersModal({
                                   }));
                                 });
                               }}
+                              dataSbId="opne_channel_setting_banned_user_context_menu_unban"
```
Comment (sravan-s) [inline src/modules/OpenChannelSettings/components/OperatorUI/BannedUsersModal.tsx:107]: opne_ > typo

**Pull request merged**